tests/api/circulation/test_loan_bulk_extend.py

```diff
@@ -0,0 +1,176 @@
+
+    day_range_loan_extendable = \
+        current_app.config["ILS_CIRCULATION_LOAN_WILL_EXPIRE_DAYS"]
+    now = arrow.utcnow()
+    end_date_not_expired = now + timedelta(days=day_range_loan_extendable - 1)
+
+    patron_id = str(users[patron_username].id)
+
+    def loan_generator(pid_value):
+        loan = {
+            "document_pid": "docid-1",
+            "end_date": end_date_not_expired.format(date_format),
+            "item_pid": {
+                "type": "pitmid",
+                "value": f"itemid-{pid_value}"
+            },
+            "patron_pid": patron_id,
+            "pickup_location_pid": "locid-1",
+            "pid": f"loanid-{pid_value}",
+            "start_date": now.format(date_format),
+            "state": item_loan_state,
+            "transaction_location_pid": "locid-1",
+            "request_start_date": "2018-06-28",
+            "transaction_user_pid": patron_id,
+            "request_expire_date": "2019-06-28",
+            "patron": {
+                "email": "patron1@test.ch"
+            }
+        }
+
+        return loan
+
+    for number in range(start, end):
+        if len(extendable_loans) < amount_of_extendable_loans:
+            extendable_loans.append(loan_generator(number))
+
+    return extendable_loans
+
+
+def create_items(start=pid_start_value, end=pid_end_value):
+    items = []
+
+    def items_generator(pid_value):
+        item = {
+            "pid": f"itemid-{pid_value}",
+            "created_by": {"type": "script", "value": "demo"},
+            "barcode": f"123456{pid_value}89-1",
+            "document_pid": "docid-1",
+            "internal_location_pid": "ilocid-1",
+            "circulation_restriction": "NO_RESTRICTION",
+            "medium": "NOT_SPECIFIED",
+            "status": "CAN_CIRCULATE",
+            "document": {}
+        }
+        return item
+
+    for number in range(start, end):
+        items.append(items_generator(number))
+
+    return items
+
+
+def index_and_commit_data(db, users, repeated_run_number=0):
+    added_value = \
+        (repeated_run_number
+         *
+         amount_of_extendable_loans
+         +
+         repeated_run_number)
+
+    new_pid_start_value = pid_start_value + added_value
+    new_pid_end_value = pid_end_value + added_value
+
+    bulk_index_items = _create_records(db, create_items(new_pid_start_value,
+                                                        new_pid_end_value),
+                                       Item, ITEM_PID_TYPE)
+    bulk_index_loans = _create_records(db, create_loans(users,
+                                                        new_pid_start_value,
+                                                        new_pid_end_value),
+                                       Loan,
+                                       CIRCULATION_LOAN_PID_TYPE)
+
+    for item in bulk_index_items:
+        RecordIndexer().index(item)
+
+    for loan in bulk_index_loans:
+        RecordIndexer().index(loan)
```

Comment:
```suggestion
        current_circulation.loan_indexer().index(loan)
```
or
`LoanIndexer().index(...)`